downloader/src/main/kotlin/vcs/Subversion.kt

```diff
@@ -112,9 +114,13 @@ class Subversion : VersionControlSystem() {
         }
 
     override fun isApplicableUrlInternal(vcsUrl: String) =
-        runCatching {
-            clientManager.wcClient.doInfo(SVNURL.parseURIEncoded(vcsUrl), SVNRevision.HEAD, SVNRevision.HEAD)
-        }.isSuccess
+        try {
+            SVNRepositoryFactory.create(SVNURL.parseURIEncoded(vcsUrl)).checkPath("", -1) != SVNNodeKind.NONE
+        } catch (e: SVNException) {
+            log.debug { "An exception was thrown when checking $vcsUrl for a $type repository: $e" }
```

Comment:
Consider putting $vcsUrl into quotes.
Would it make sense to log the full stack trace?

Reply:
We usually do not quote URLs (as they cannot contain spaces, in contrast to e.g. file paths), so I also didn't do it here. But I'm calling `e.showStackTrace()` now.